### PR TITLE
Upgrade to bootstrap-vue 2.5.0

### PR DESCRIPTION
**Pull request by openorclose**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• Enhancement to an existing feature

**What is the rationale for this request?**

Bootstrap-vue 2.5.0 has much better popovers and tooltips. They allow you to hover over the popover/tooltip and it won't disappear.

**What changes did you make? (Give an overview)**

Updated bootstrap-vue js and css

**Provide some example code that this change will affect:**

```js

```

**Is there anything you'd like reviewers to focus on?**

Nothing else is broken

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**

### CODE REVIEW

**Comment by ang-zeyu** [comment]
The [changelog](https://bootstrap-vue.js.org/docs/misc/changelog/) dosen't seem to have breaking changes either.

The positioning still seems to be off for top/bottom though, as mentioned in https://github.com/MarkBind/markbind/pull/1033#issuecomment-591869641
positioning

**Comment by openorclose** [comment]
> The [changelog](https://bootstrap-vue.js.org/docs/misc/changelog/) dosen't seem to have breaking changes either.
> 
> The positioning still seems to be off for top/bottom though, as mentioned in [#1033 (comment)](https://github.com/MarkBind/markbind/pull/1033#issuecomment-591869641)
> positioning

Oh right, let me investigate

File: package.json [hunk 1]
```diff
@@ -69,9 +69,9 @@
     "jest": "jest",
     "lint": "./node_modules/.bin/eslint .",
     "lintfix": "./node_modules/.bin/eslint . --fix",
-    "pretest": "npm run lint && npm run csslint",
+    "pretest": "",
     "pretestwin": "npm run lint && npm run csslint",
-    "test": "jest && cd test/functional && ./test.sh",
+    "test": "cd test/functional && ./test.sh",
```
Comment (ang-zeyu) [inline package.json]: What are these for?

Comment (ang-zeyu) [reply package.json]: The positioning seems to be fixed now otherwise 👍

Comment (openorclose) [reply package.json]: Oops, missed this out.

Comment (openorclose) [reply package.json]: Fixed!

**Comment by yamgent** [comment]
Where did you obtain the `min.js` from? My copy of `bootstrap-vue@2.5.0` does not match yours.

**Comment by openorclose** [comment]
> Where did you obtain the `min.js` from? My copy of `bootstrap-vue@2.5.0` does not match yours.

I got it from https://unpkg.com/bootstrap-vue@2.5.0/dist/bootstrap-vue.min.js

**Comment by marvinchin** [comment]
> Where did you obtain the min.js from? My copy of bootstrap-vue@2.5.0 does not match yours.

> I got it from https://unpkg.com/bootstrap-vue@2.5.0/dist/bootstrap-vue.min.js

@yamgent is this good to go?

**Pull request merged**